src/commands/test.js

```diff
@@ -53,16 +53,19 @@ function getPlatform() {
   const type = os.type();
   const arch = os.arch();
   const release = os.release();
+  const cpuCore = os.cpus()[0];
   const majorVersion = release.substr(0, release.indexOf('.'));
 
-  if (arch === 'x64') {
+  if (arch === 'x64' || (arch === 'arm64' && cpuCore.model.includes("M1"))) {
```

Comment:
I think it's better to add the prefix `Apple `. I say that because it may happen that a CPU has a model with `M1` in their name.

Reply:
I see what you've done here in the second part of this `if` statement and I'm okay with keeping it, but I don't think it gets into this branch because of it.

Example output from my M1 Mac:
![スクリーンショット 2021-10-18 20 40 11](https://user-images.githubusercontent.com/15306309/137820748-ed60d87a-54c8-48f7-a90b-ec89aa5f7b45.png)

As far as I know Node.js can only run via Intel emulation, that's why it always shows up `x64` in `os.arch()`. However, once they support it, this `||` part will probably work as well.

No need to change, just detailing this for the future 🙂

Reply:
I have pushed the changes. It is actually already possible to have `os.arch()` return `arm64` on an M1 Mac, it happened on the mac I was testing it on.

Reply:
Oh really? Awesome then 🙂